actions/databases.go

```diff
@@ -201,50 +199,52 @@ func DatabasesPost(c buffalo.Context) error {
-		pg, ok := rdsClient.DefaultDBParameterGroupName[pgFamily]
-		if !ok {
-			log.Println("No matching DefaultDBParameterGroupName found in config, using AWS default PG")
-		} else {
-			log.Println("Using DefaultDBParameterGroupName:", pg)
-			input.Instance.DBParameterGroupName = aws.String(pg)
+	// create rds instance, if specified
+	if input.Instance != nil {
+		// set default subnet group
+		if input.Instance.DBSubnetGroupName == nil {
+			input.Instance.DBSubnetGroupName = aws.String(rdsClient.DefaultSubnetGroup)
 		}
-	}
-
-	input.Instance.Tags = normalizeTags(input.Instance.Tags)
-	if instanceOutput, err = rdsClient.Service.CreateDBInstanceWithContext(c, input.Instance); err != nil {
-		log.Println(err.Error())
-		if input.Cluster != nil {
-			// if this instance was in a new cluster, delete the cluster
-			log.Println("Deleting cluster", *input.Cluster.DBClusterIdentifier)
-			clusterInput := &rds.DeleteDBClusterInput{
-				DBClusterIdentifier: input.Cluster.DBClusterIdentifier,
-				SkipFinalSnapshot:   aws.Bool(true),
+		// set default parameter group
+		if input.Instance.DBParameterGroupName == nil {
+			pgFamily, pgErr := rdsClient.DetermineParameterGroupFamily(input.Instance.Engine, input.Instance.EngineVersion)
+			if pgErr != nil {
+				log.Println(pgErr.Error())
+				return c.Error(400, pgErr)
 			}
-			if _, errc := rdsClient.Service.DeleteDBClusterWithContext(c, clusterInput); errc != nil {
-				log.Println("Failed to delete cluster", errc.Error())
+			log.Println("Determined ParameterGroupFamily based on Engine:", pgFamily)
+			pg, ok := rdsClient.DefaultDBParameterGroupName[pgFamily]
+			if !ok {
+				log.Println("No matching DefaultDBParameterGroupName found in config, using AWS default PG")
 			} else {
-				log.Println("Successfully requested deletion of cluster", *input.Cluster.DBClusterIdentifier)
+				log.Println("Using DefaultDBParameterGroupName:", pg)
+				input.Instance.DBParameterGroupName = aws.String(pg)
 			}
 		}
-		if aerr, ok := err.(awserr.Error); ok {
-			return c.Error(400, aerr)
+
+		input.Instance.Tags = normalizeTags(input.Instance.Tags)
+		if instanceOutput, err = rdsClient.Service.CreateDBInstanceWithContext(c, input.Instance); err != nil {
+			log.Println(err.Error())
+			if input.Cluster != nil {
+				// if this instance was in a new cluster, delete the cluster
+				log.Println("Deleting cluster", *input.Cluster.DBClusterIdentifier)
+				clusterInput := &rds.DeleteDBClusterInput{
+					DBClusterIdentifier: input.Cluster.DBClusterIdentifier,
+					SkipFinalSnapshot:   aws.Bool(true),
+				}
+				if _, errc := rdsClient.Service.DeleteDBClusterWithContext(c, clusterInput); errc != nil {
+					log.Println("Failed to delete cluster", errc.Error())
+				} else {
+					log.Println("Successfully requested deletion of cluster", *input.Cluster.DBClusterIdentifier)
+				}
+			}
+			if aerr, ok := err.(awserr.Error); ok {
+				return c.Error(400, aerr)
```

Comment:
Maybe you can convert this to use rollback since that's basically what its doing.  Should be pretty easy.